cassandra-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/jmx/NodeRepairJob.java

```diff
@@ -83,18 +67,16 @@ public boolean repairNextKeyspace() {
             commandToKeyspace.put(commandNo, keyspace);
 
             LOGGER.info("Submitted repair for keyspace {} with cmd#{}", keyspace, commandNo);
-            return true;
+            break;
         }
     }
 
-    private void finished() {
+    protected void finished() {
         try {
-            finishedTimestamp = System.currentTimeMillis();
-            long duration = System.currentTimeMillis() - startTimestamp;
-            LOGGER.info("Repair job finished in {} seconds : {}", duration / 1000L, keyspaceStatus);
+            super.finished();
             jmxConnect.getStorageServiceProxy().removeNotificationListener(this);
             close();
-        } catch (ListenerNotFoundException ignored) {
+        } catch (Exception ignored) {
```

Comment:
Please document why you are swallowing all exceptions rather than enumerating the exceptions that are valid to swallow.